examples/src/bin/texture.rs

```diff
@@ -264,7 +264,8 @@ fn main() {
         let image = image::load_from_memory(include_bytes!("../../assets/rust.png"))
             .unwrap()
             .to_rgba8();
-        let image_dimensions = image.dimensions();
+        let (width, height) = image.dimensions();
+        let image_dimensions = vk::Extent2D { width, height };
```

Comment:
rename local variable to `image_extent2D` ?

Reply:
I've renamed it to `image_extent`... `image_extent2D` violates my understanding of naming conventions due to the uppercase `D`. I'd be ok with `image_extent_2d` though my preference is to omit the `_2d`.